ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/PullPhysicalPlan.java

```diff
@@ -164,7 +164,8 @@ public QueryId getQueryId() {
     // Could be one or more keys
     KEY_LOOKUP,
     RANGE_SCAN,
-    TABLE_SCAN
+    TABLE_SCAN,
+    UNKNOWN
```

Comment:
We cannot know what the WHERE clause is for stream pull queries since it's not analyzed by the logical plan